src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala

```diff
@@ -97,8 +97,15 @@ case class DepositDir private(baseDir: File, user: String, id: UUID) extends Deb
   /**
    * Deletes the deposit.
    */
-  def delete(): Try[Unit] = Try {
-    bagDir.parent.delete()
+  def delete(): Try[Unit] = {
+    val states = Seq(State.draft, State.archived, State.rejected)
```

Comment:
Perhaps move this list to the `State` enum? I don't like it being created every time you call this method.